### ISSUE TITLE
Adding any Command Line Interface when running my OSX tests throw error: unknown option

### ISSUE THREAD

**Issue by bennymeade** [rquellh/testcafe-cucumber#22]
Hello, hope somebody can help.
I have successfully been running all my TestCafe Cucumber tests on OSX with: ./node_modules/.bin/cucumber-js

But I cannot add any Command Line Interface when running my tests, as they are not recognised. 
Example:
$ ./node_modules/.bin/cucumber-js --skip-js-errors
error: unknown option `--skip-js-errors'

In fact I cannot seem to use any commands from the docs, because of the same 'error: unknown option': http://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html

What am I doing wrong?

**Comment by maxharding4** [comment]
Hi @bennymeade , did you ever find a work around for this?

Cheers,

**Comment by bennymeade** [comment]
@maxharding4 Nope, for the 'js-errors' our developers removed those errors. I commonly use these run commands now:

$ ./node_modules/.bin/cucumber-js --world-parameters '{"browser": "chrome"}'
$ ./node_modules/.bin/cucumber-js --world-parameters '{"browser": "chrome:headless"}'
$ ./node_modules/.bin/cucumber-js --name "Cucumber JS Scenario Name"
$ ./node_modules/.bin/cucumber-js features/homePage.feature
$ ./node_modules/.bin/cucumber-js --name "Cucumber JS Scenario Name" --world-parameters '{"browser": "chrome:headless"}'
etc

**Comment by maxharding4** [comment]
Thanks for replying @bennymeade, will probably be a while until the errors are removed from the page I'm looking at!
